src/main/java/com/minecolonies/coremod/blocks/BlockTimberFrame.java

```diff
@@ -77,6 +77,7 @@ public void registerItemBlock(final IForgeRegistry<Item> registry)
      */
     @NotNull
     @Override
+    @Deprecated
     public MapColor getMapColor(@NotNull final IBlockState state)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1133)